tests/unit/test_utils.py

```diff
@@ -18,11 +23,49 @@
     "protocol_mapper",
     "protocols.json",
     "config",
-    "utils",
 ]
 
 _EXCLUDED_DECORATOR_FUNCTIONS = ["wraps", "total_ordering", "abstractmethod"]
 
+_EXCLUDED_FUNCTIONS = ["jinja2_convenience_function", "import_module"]
+
+UPTIME_TO_STRING = [
+    {
+        "sent": 604800,
+        "received": "1 week",
+    },
+    {
+        "sent": 86400,
+        "received": "1 day",
+    },
+    {
+        "sent": 7200,
+        "received": "2 hours",
+    },
+    {
+        "sent": 3600,
+        "received": "1 hour",
+    },
+    {
+        "sent": 7250,
+        "received": "2 hours, 50 seconds",
+    },
+    {
+        "sent": 698490,
+        "received": "1 week, 1 day, 2 hours, 1 minute, 30 seconds",
+    },
+]
+
+UPTIME_TO_SECONDS = [
+    {"sent": "1 week, 0 days, 0 hours, 10 minutes", "received": 605400},
```

Comment:
Make sure to have years included in the tests as well.

Reply:
Updated